### PR TITLE
ODC-7670: Migrate MoveConnectionModal to PF5

### PR DESCRIPTION
**Pull request by logonoff**
**Fixes**: 

https://issues.redhat.com/browse/ODC-7670

**Screen shots / Gifs for design review**: 

demo:

https://github.com/user-attachments/assets/0fd4ea88-b89d-48f0-9edb-d1ad0b976e22

**Unit test coverage report**: 

n/a

**Test setup:**

add some pods and create a visual connector on the topology page

**Browser conformance**: 

- [x] Chrome
- [x] Firefox
- [ ] Safari

### CODE REVIEW

**Comment by logonoff** [comment]
Tech debt that is not customer facing, so manually adding labels
/label px-approved
/label docs-approved

**Review by vikram-raj — CHANGES_REQUESTED** [review]
Clicking anywhere in the window does not close the dropdown option. It should close the dropdown. 

https://github.com/user-attachments/assets/a3f99f7e-072a-42df-85bd-2b9de686314e

File: frontend/packages/topology/src/components/modals/MoveConnectionModal.tsx [hunk 1]
```diff
@@ -98,21 +93,30 @@ const MoveConnectionForm: React.FC<
...
-                  {nodeItem(values.target)}
-                </DropdownToggleDeprecated>
-              }
+              onSelect={(_, value: Node) => {
+                if (value) {
+                  values.target = value;
+                }
+                setOpen(false);
+              }}
+              toggle={toggle}
```
Comment (vikram-raj) [inline frontend/packages/topology/src/components/modals/MoveConnectionModal.tsx:105]: Need to add prop `onOpenChange={(open) => setOpen(open)}` to close the options menu when clicking anywhere in the window.

Comment (logonoff) [reply frontend/packages/topology/src/components/modals/MoveConnectionModal.tsx:105]: amended, thanks for catching :)

**Review by vikram-raj — APPROVED** [review]
/lgtm

**Comment by The-Anton** [comment]
Verified on cluster, working fine
/label qe-approved

Screenshot 2024-09-03 at 3 02 24 PM

**Pull request merged**